TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -15,6 +15,29 @@ public CodeElement CreateCommitStatement(CodeElementsParser.CommitStatementConte
             return new CommitStatement();
         }
 
+        public CodeElement CreateRollbackStatement(CodeElementsParser.RollbackStatementContext context)
+        {
+            SavePointClause savePointClause = null;
+            if (context.savePointClause() != null)
+            {
+                savePointClause = CreateSavePointClause(context.savePointClause());
+            }
+            return new RollbackStatement(savePointClause);
+        }
+
+        private SavePointClause CreateSavePointClause(CodeElementsParser.SavePointClauseContext context)
+        {
+            SymbolReference savePointName = null;
+            if (context.savePoint_name != null)
+            {
+                Token savePointToken = context.savePoint_name as Token;
+                savePointName= new SymbolReference(new AlphanumericValue(savePointToken), SymbolType.SqlIdentifier);
```

Comment:
```suggestion
                savePointName= new SymbolReference(new AlphanumericValue((Token) context.savePoint_name), SymbolType.SqlIdentifier);
```
Better use a direct cast `(Token)` because here, we are sure that the cast will succeed.